eclair-core/src/test/scala/fr/acinq/eclair/channel/states/c/WaitForDualFundingConfirmedStateSpec.scala

```diff
@@ -164,7 +164,6 @@ class WaitForDualFundingConfirmedStateSpec extends TestKitBaseClass with Fixture
     import f._
     val fundingTx = alice.stateData.asInstanceOf[DATA_WAIT_FOR_DUAL_FUNDING_CONFIRMED].latestFundingTx.sharedTx.asInstanceOf[FullySignedSharedTransaction].signedTx
     alice ! WatchPublishedTriggered(fundingTx)
-    assert(alice2blockchain.expectMsgType[WatchFundingSpent].txId == fundingTx.txid)
```

Comment:
We should verify that we set a `WatchFundingConfirmed` and that we don't set `WatchFundingSpent` (same thing for the non-initiator test):

```scala
assert(alice2blockchain.expectMsgType[WatchFundingConfirmed].txId == fundingTx.txid)
alice2blockchain.expectNoMessage(100 millis) // we don't set WatchFundingSpent
```

Reply:
Done in 6e63a7623.